modules/installing-odh-components.adoc

```diff
@@ -0,0 +1,63 @@
+:_module-type: PROCEDURE
+//pv2hash: 
+
+[id='installing-odh-components_{context}']
+= Installing Open Data Hub components
+
+[role='_abstract']
+You can use the OpenShift web console to install specific components of Open Data Hub on your cluster when version 2.X of the Open Data Hub Operator is already installed on the cluster.
+
+.Prerequisites
+* You have installed v2.X of the Open Data Hub Operator.
+* You can log in as a user with `cluster-admin` privileges.
+* If you want to use the `kserve`, `datasciencepipeline`, `distributedworkloads`, or `modelmesh` components, you must have already installed the following Operator or Operators for the component. For information about installing an Operator, see link:https://access.redhat.com/documentation/en-us/openshift_container_platform/4.14/html/operators/administrator-tasks#olm-adding-operators-to-a-cluster[Adding Operators to a cluster].
+
+.Required Operators for components
+[cols="3]
+|===
+| Component | Required Operators | Catalog
+
+| ksersve
+| Red Hat OpenShift Serverless Operator, Red Hat OpenShift Service Mesh Operator 
+| Red Hat
+
+| datasciencepipeline
+| Red Hat OpenShift Pipelines Operator
+| Red Hat
+
+| distributedworkloads
+| CodeFlare Operator
+| Community
+
+| modelmesh
+| Prometheus Operator
+| Community
+
+|===
+
```

Comment:
add another prerequisite item for trusty ai:
 
 * If you want to use the `trustyai` component, you must enable user workload monitoring as described in link:https://docs.openshift.com/container-platform/4.13/monitoring/enabling-monitoring-for-user-defined-projects.html[Enabling monitoring for user-defined projects].